spynnaker/pyNN/models/common/abstract_spike_recordable.py

```diff
@@ -45,3 +56,14 @@ def get_spikes(
         :return: A numpy array of 2-element arrays of (neuron_id, time)\
                 ordered by time
         """
+
+    @abstractmethod
+    def get_spikes_sampling_interval(self):
+        """ Returns the current sampling interval for spikes
+        :return: Sampling interval in micro seconds
+
+        if self.is_recording_spikes():
```

Comment:
Is the content here meant to be commented out?

Reply:
Yes or better yet just deleted!  It is an abstractmethod after all.